modules/core/src/main/scala/advxml/syntax/AllTransformSyntax.scala

```diff
@@ -60,11 +60,11 @@ private[syntax] sealed trait RuleSyntax {
 
 private[syntax] sealed trait ZoomSyntax {
 
-  implicit class XmlZoomNodeBaseOps[Z <: XmlZoomNodeBase](zoom: Z) {
+  implicit class XmlZoomNodeBaseOps[Z <: XmlZoomNodeBase](val zoom: Z) {
```

Comment:
Why `zoom` should be a field of the class ?

Reply:
It has to be public because it’s visible (now that it is not from a projection) in the return type of the operators.

Reply:
Ok clear 👍